todo.js

```diff
@@ -0,0 +1,112 @@
+import { v4 as uuidv4 } from 'uuid'
+
+export class Todo {
+    #id
+    #isDone
+    #text
+
+    constructor(text) {
+        this.#id = uuidv4()
+        this.#isDone = false
+        this.#text = text
+    }
+
+    getId() {
+        return this.#id
+    }
+
+    getIsDone() {
+        return this.#isDone
+    }
+
+    setIsDone(value) {
+        this.#isDone = value
+    }
+
+    setText(value) {
+        this.#text = value
+    }
+
+    render(onClick, todos, taskName) {
+        const newDiv = document.createElement('div')
+        const newListItem = this.#createNewListItem(onClick)
+        const newSpan = this.#createSpanBtns(todos)
+
+        newDiv.classList.add('list-item')
+        newListItem.classList.add('list-item__li-tag')
+        newListItem.textContent = taskName.value
```

Comment:
this.#text?

Reply:
this code sets a textContent of DOM element to value given by input, it's not changes the Todo object

Reply:
yes I know, but you don't need to pass `taskName` via argument. You already have todo's text inside `this.#text`

Reply:
ok, nvm, I fixed it ;)